test/e2e/sauce.wdio.conf.ts

```diff
@@ -24,27 +24,7 @@ exports.config = {
   maxInstances: 1,
   waitforTimeout: 15000,
   specs: [
-    path.resolve(__dirname, 'features/**/interaction-code-flow.feature')
-  ],
-  capabilities: [
-    {
-      maxInstances: 1, // all tests use the same user and local storage. they must run in series
-      browserName: 'MicrosoftEdge',
-      browserVersion: 'latest',
-      platformName: 'Windows 10',
-      "ms:edgeOptions": {} // don't delete this line, edge tests won't run
-    },
-    {
-      maxInstances: 1, // all tests use the same user and local storage. they must run in series
-      browserName: 'internet explorer',
-      browserVersion: 'latest',
-      platformName: 'Windows 10',
-      "se:ieOptions": {
-          acceptUntrustedCertificates: true,
-          "ie.ensureCleanSession": true
-      },
-      timeouts: { "implicit": 20_000 }
-    }
+    path.resolve(__dirname, 'features/**/widget-flows.feature')
```

Comment:
This test which loads the widget in different states is a better test on IE11, Edge and iOS. We want to ensure widget loads on these browsers